### ISSUE TITLE
Runing QnASample-0 with no keys QnA key can go on for ever needs time out

### ISSUE THREAD

**Issue by MoesLab** [microsoft/BotFramework-Composer#5424]
## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

**Comment by joshgummersall** [comment]
@MoesLab, would you please fill out the issue template with more details? In particular, a specific set of steps to reproduce the issue. Thanks!

**Comment by tonyanziano** [comment]
Closing due to inactivity. Please feel free to reopen after providing the details described above by @joshgummersall

**Comment by MoesLab** [comment]
I have deleted the resources, I will start it again and take notes. if the
problem persists I will let you know.
Regards,

**Comment by MoesLab** [comment]
Is there any video showing how to set up and connect a bot created by the
composer app (installed on any PC) to resources particularly with QnA and
LUIS and publish it to Azure. not only shows how to create and modify
intents?
I don't have a clear view of all prerequisites needed for all these steps,
it looks there are multiway to do the same thing.
I understand the whole thing is under development.
I have found some discrepancies in naming and labeling that cause confusion
for newbies.
I have found once a person steps into the development part he/she will step
into multiple choices and branches which makes the process very painful for
some.
I have found that constant upgrades and improvement move and changes thing
around that videos and information become confusing in some cases.
basically, I am looking for a HOW TO video/doc that allows me to set up a
chatbot with QnA and LUIS very quickly without installing prerequisites
such as Yarn and other CLI, likely from the Azure portal itself.
can you direct me to the right sources?

Regards,
